### PR TITLE
Add inorganic_path_for

### PR DESCRIPTION
**Pull request by luchotc**
## Goal

Add `inorganic_path_for` method which returns the request path without the organization part.

### CODE REVIEW

File: lib/mumukit/platform/organization_mapping.rb [hunk 1]
```diff
@@ -60,5 +68,9 @@ def self.organic_uri(uri, organization)
     def self.path_under_namespace?(organization_name, path, namespace)
       path.start_with? "/#{organization_name}/#{namespace}/"
     end
+
+    def self.inorganic_path_for(request)
+      path_for(request).split('/').drop(2).join('/')
```
Comment (faloi) [inline lib/mumukit/platform/organization_mapping.rb:73]: I found this one that seems nicer:

```rb
require 'pathname'

def split_path(path)
    Pathname(path).each_filename.to_a
end
```

File: lib/mumukit/platform/organization_mapping.rb [hunk 2]
```diff
@@ -49,8 +65,17 @@ def self.map_organization_routes!(native, framework, &block)
       framework.configure_tenant_path_routes! native, &block
     end
 
+    def self.path_composition_for(request)
+      path_parts = Pathname(path_for(request)).each_filename.to_a
```
Comment (julian-berbel) [inline lib/mumukit/platform/organization_mapping.rb:69]: just nitpicking, but you could avoid the `delete_at(0)` later by splitting here
`orga, *path_parts = Pathname(path_for(request)).each_filename.to_a`

**Review by julian-berbel — APPROVED** [review]
Really, it's the same either way.

Looks good!

File: lib/mumukit/platform/organization_mapping.rb (hunk 2 above)
Comment (luchotc) [reply lib/mumukit/platform/organization_mapping.rb]: I'll change it as it's much cleaner that way

**Pull request merged**